src/Http/Controllers/Character/KillmailController.php

```diff
@@ -24,7 +24,7 @@
 
```

Comment:
# Character Killmail
- sorting by victim is throwing an exception
```
DataTables warning: table id=character-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'character_name' in 'order clause' (SQL: select *, `character_killmails`.`character_id` as `ownerID`, `killmail_victims`.`character_id` as `victimID` from `character_killmails` left join `killmail_details` on `character_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `character_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `character_killmails`.`character_id` = 90795931 order by `character_name` asc limit 100 offset 0)
```

- sorting by ship type is throwing an exception
```
DataTables warning: table id=character-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type_name' in 'order clause' (SQL: select *, `character_killmails`.`character_id` as `ownerID`, `killmail_victims`.`character_id` as `victimID` from `character_killmails` left join `killmail_details` on `character_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `character_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `character_killmails`.`character_id` = 90795931 order by `type_name` asc limit 100 offset 0)
```

- sorting by location is throwing an exception
```
DataTables warning: table id=character-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'item_name' in 'order clause' (SQL: select *, `character_killmails`.`character_id` as `ownerID`, `killmail_victims`.`character_id` as `victimID` from `character_killmails` left join `killmail_details` on `character_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `character_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `character_killmails`.`character_id` = 90795931 order by `item_name` desc limit 100 offset 0)
```

- making a reasearch is throwing an exception
```
DataTables warning: table id=character-killmails - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'character_killmails.killmail_time' in 'where clause' (SQL: select count(*) as aggregate from (select '1' as `row_count` from `character_killmails` left join `killmail_details` on `character_killmails`.`killmail_id` = `killmail_details`.`killmail_id` left join `killmail_victims` on `killmail_victims`.`killmail_id` = `character_killmails`.`killmail_id` left join `invTypes` on `killmail_victims`.`ship_type_id` = `invTypes`.`typeID` left join `mapDenormalize` on `killmail_details`.`solar_system_id` = `mapDenormalize`.`itemID` where `character_killmails`.`character_id` = 90795931 and (LOWER(`character_killmails`.`killmail_time`) LIKE %g% or LOWER(`character_killmails`.`character_name`) LIKE %g% or LOWER(`character_killmails`.`type_name`) LIKE %g% or LOWER(`character_killmails`.`item_name`) LIKE %g%)) count_row_table)
```